sympy/core/evalf.py

```diff
@@ -336,9 +336,9 @@ def calc_part(expr, nexpr):
     re, im, re_acc, im_acc = None, None, None, None
 
     if ire:
-        re, re_acc = calc_part(C.re(expr, evaluate=False), ire)
+        re, re_acc = calc_part(sympy.re(expr, evaluate=False), ire)
```

Comment:
The computed re and im are just recomputed real and imaginary parts of the expr. I think it would be fine to just name `re, im, re_acc, im_acc` -> `xre, xim, xre_acc, xim_acc`. I just feel better about explicitly importing `re` and `im` rather than relying on the fact that they have been imported into the `sympy` namespace. I suppose it's partly the precedent, too, to use `sympy` like `C` has been used. It's based more on feeling than logic; you would have a sense (ahving made these changes) whether there is any intuition driving that feeling.